packages/@vue/cli-plugin-eslint/index.js

```diff
@@ -52,7 +52,7 @@ module.exports = (api, options) => {
           resolveModule('eslint/package.json', cwd) ||
             resolveModule('eslint/package.json', __dirname)
         ),
-        formatter: loadModule('eslint/lib/formatters/codeframe', cwd, true)
+        formatter: loadModule('eslint/lib/cli-engine/formatters/codeframe', cwd, true)
```

Comment:
```suggestion
        formatter: 'codeframe'
```

Since we have specified the path of `eslint` via `eslintPath`,  using [`eslint.loadFormatter('codeframe')`](https://github.com/webpack-contrib/eslint-webpack-plugin/blob/2e6c8eb62d50c3bdca7a6bd33b4acefacdccf402/src/linter.js#L238) can get the correct formatter, and more robust. What do you think about it?

Reply:
I think `codeframe` is more nature as eslint offical document provide this. using function it look like we are using custom formatter